domain/src/memory.js

```diff
@@ -86,6 +88,17 @@ function selectNotDeleted(dict) {
 }
 
 /**
+ * Полностью обновляет предмет(кроме id)
+ * @returns {Promise.<Number, Error>} Количество найденных объектов
+ */
+async function updateItemFully(db, coll, id, item) {
```

Comment:
I think the name of the function is misleading. What does it mean "fully"? Why don't we have `addItemFully` then? Make it more specific, ie `updateExistingItem`, or consistent with other functions, ie `updateItem`.